components/generic/PaginationNav.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <b-pagination-nav
+    v-model="currentPage"
+    :limit="limit"
+    :hide-ellipsis="hideEllipsis"
+    :number-of-pages="totalPages"
+    base-url="#page="
```

Comment:
I think we want to be using [router links](https://bootstrap-vue.js.org/docs/components/pagination-nav/#router-links) here with parameters not after the `#`.